poetry/utils/env.py

```diff
@@ -348,9 +347,10 @@ def get(self, reload=False):  # type: (bool) -> Env
         if not in_venv or env is not None:
             # Checking if a local virtualenv exists
             if (cwd / ".venv").exists() and (cwd / ".venv").is_dir():
-                venv = cwd / ".venv"
+                if self._poetry.config.get("virtualenvs.in-project"):
```

Comment:
Should we handle `None` as "use detected environment"?

Reply:
Didn't know that we can set it to `None` :) As @sdispater suggested in https://github.com/python-poetry/poetry/issues/1770#issuecomment-589642728 I could change it to detect `.venv` in case `in-project` is set to `true` or `None`. But than I guess I will update the docs as well, to the behavior clear.

Reply:
It's the case when it was not explicitly set right? Unless we ensure that not set means true by default.